lib/types.ts

```diff
@@ -41,6 +41,7 @@ export interface PluginMetadata {
   dockerImageId: string;
   imageLayers: string[];
   rootFs?: string[];
+  platform?: string;
```

Comment:
Just an FYI that as part of #project-envelope unfortunately the `PluginMetadata` no longer exists in the response from the plugin, so all the fields in this interface will find a new home 😄 https://gist.github.com/ivanstanev/7f894d9bc3e4611452eadce1fe2c6729#file-scan-ts-L13-L16

Is it possible to put it for example in `scannedProject[0].meta` as this will most likely be the place we will use in envelope?

Reply:
Great, thanks! 👍

Reply:
on it 👍